kingpin/actors/base.py

```diff
@@ -132,10 +132,17 @@ def _validate_options(self):
         """
 
         # Loop through all_options, and find the required ones
-        # Required options have `None` as their default value.
+        self.log.debug(self.all_options.items())
+        if len(self.all_options.items()):
```

Comment:
Whats going on here? The code is simple -- but I don't actually know what its doing.